tests/data/arxiv_api_response.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Seems like this contains real data. Another approach would be to just fill all the entries with "fake" text like we did here for example https://github.com/BlueBrain/Search/blob/master/tests/data/jats_article.xml. However, it is your call.

Reply:
Removed the titles, authors, abstract contents in 87f4494.